src/main/resources/application.properties

```diff
@@ -1 +1,7 @@
-quarkus.http.port=8081
\ No newline at end of file
+quarkus.http.port=8081
+quarkus.datasource.url = jdbc:postgresql://localhost:5432/quarkus
```

Comment:
When trying to run the application I get the following error:
```
[error]: Build step io.quarkus.agroal.deployment.AgroalProcessor#build threw an exception: io.quarkus.runtime.configuration.ConfigurationException: quarkus.datasource.url and quarkus.datasource.driver have been deprecated in Quarkus 1.3 and removed in 1.9. Please use the new datasource configuration as explained in https://quarkus.io/guides/datasource.
```
So please change the corresponding properties